apicast/src/oauth/keycloak.lua

```diff
@@ -27,6 +27,12 @@ _M.params = {
   }
 }
 
+local function timestamp_to_seconds_from_now(expiry)
+  local time_now = os.time()
```

Comment:
`ngx.now()` should be used as it is more performant and does not include a syscall


also the case when expiry is nil should be handled